_pages/robot/2019.html

```diff
@@ -50,5 +50,10 @@ <h5>Programming</h5>
         <li><a href="https://github.com/frc1418/VictiScout">VictiScout scouting system</a></li>
     </ul>
     <h5>Competition Performance</h5>
-    <p>We will be competing in two competitions this year at Battlefield HS (March 1-3) and at Oxon Hill HS (March 22-24). You can find out more about our competition performance and schedule on <a href="https://www.thebluealliance.com/team/1418">our Blue Alliance page</a>, and look out for videos about our competitions on our <a href="https://www.youtube.com/user/Robotics1418">YouTube page</a>.</p>
+    <p>We will be competing in two competitions this year at Battlefield HS (March 1-3) and at Oxon Hill HS (March 22-24). You can find out more about our competition performance and schedule on <a href="https://www.thebluealliance.com/team/1418">our Blue Alliance page</a>.</p>
+    <div class="iframe-ext-wrapper">
+      <div class="iframe-wrapper">
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/KtN70-pMvhw" allowfullscreen></iframe>
```

Comment:
Just an idea for the future, it could be neat to set up a Jekyll `include` for videos like this